packages/rancher-monitoring/overlay/templates/rancher-monitoring/clusterrole.yaml

```diff
@@ -105,21 +90,32 @@ rules:
   - 'get'
   - 'list'
   - 'watch'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: monitoring-config-admin
+  labels: {{ include "kube-prometheus-stack.labels" . | nindent 4 }}
+  {{- if .Values.global.rbac.userRoles.aggregateToDefaultRoles }}
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+  {{- end }}
```

Comment:
does it make sense for these to also be aggregated into the default k8s roles?

Reply:
Based on https://github.com/rancher/charts/pull/648#discussion_r491640992, this will be removed in the next commit